docs/Configuration/yaml-files.md

```diff
@@ -352,7 +406,7 @@ software:
         - Marketing
 ```
 
-Use `labels_include_any` to target hosts that have any label in the array or `labels_exclude_any` to target hosts that don't have any label in the array. Only one of `labels_include_any` or `labels_exclude_any` can be specified. If neither are specified, all hosts are targeted.
+Use `labels_include_any_path` to target hosts that have any label or `labels_exclude_any_path` to target hosts that don't have any label. Only one of `labels_include_any_path` or `labels_exclude_any_path` can be specified. If neither are specified, all hosts are targeted.
```

Comment:
Maintain support for specifying an array of label names using `labels`, `labels_include_any `, and `labels_exclude_any` for backwards compatibility.

Only one of `labels`,  `labels_include_any `, `labels_exclude_any` and the new `_path` equivalents can be specified.

If more than one is specified, an error is returned and no changes are applied:
```
Error: Couldn’t edit software. For each software, only one of "labels_include_any_path" or "labels_exclude_any_path" can be included.
```